src/scss/components/_filter.scss

```diff
@@ -195,7 +194,7 @@
 	.filter {
 		.filter-header {
 			h2 {
-				font-size: rem(48);
+				font-size: rem(35);
```

Comment:
This `font-size` definition can be removed as it matches the default font size for `h2`.